boards/DCM/Inc/App/states/App_AllStates.h

```diff
@@ -21,20 +20,25 @@ static inline bool App_SharedStates_HasInverterFaulted(const struct DcmCanRxInte
  * @param can_rx_interface The CAN Rx interface to get the CAN signals from
  * @return true if the start switch is on, false otherwise
  */
-static inline bool App_SharedStates_IsStartSwitchOn(const struct DcmCanRxInterface *can_rx_interface)
+static inline bool App_IsStartSwitchOn(const struct DcmCanRxInterface *can_rx_interface)
 {
     return App_CanRx_DIM_SWITCHES_GetSignal_START_SWITCH(can_rx_interface) ==
            CANMSGS_DIM_SWITCHES_START_SWITCH_ON_CHOICE;
 }
 
+static inline bool App_IsBmsInDriveState(const struct DcmCanRxInterface *can_rx)
+{
+    return App_CanRx_BMS_STATE_MACHINE_GetSignal_STATE(can_rx) == CANMSGS_BMS_STATE_MACHINE_STATE_DRIVE_CHOICE;
+}
+
 /**
  * On-tick 1Hz function for every state in the given state machine
  * @param state_machine The state machine to run on-tick function for
  */
-void App_SharedStatesRunOnTick1Hz(struct StateMachine *state_machine);
+void App_AllStatesRunOnTick1Hz(struct StateMachine *state_machine);
 
 /**
  * On-tick 100Hz function for every state in the given state machine
  * @param state_machine The state machine to run on-tick function for
```

Comment:
add comment here

Reply:
w/ addition of bool

Reply:
For App_AllStatesRunOnTick1Hz? This doesnt return a bool

Reply:
for the 100 hz